### ISSUE TITLE
Documentation for --host-url

### ISSUE THREAD

**Issue by Abhishaike** [sokrypton/ColabFold#347]
It'd be highly appreciated if there was documentation on the specifications of the --host-url! 

I know prior responses have said to simply run colabfold_search and use the resulting .a3m file, but it'd be far more streamlined to set up our own API.

**Comment by Abhishaike** [comment]
Ah, just stumbled across this, is this up-to-date? https://github.com/sokrypton/ColabFold/blob/2b535b6c670db09051245b794c76a97c4d4da116/MsaServer/README.md

**Comment by milot-mirdita** [comment]
Yes, that's up to date. But the local template feature is still not done. I didn't have time to finish that.
